### ISSUE TITLE
Dist is full example

### ISSUE THREAD

**Issue by dvdplm** [unclecheese/react-selectable#3]
Hi there!
I'm looking to using this in our project and I'm wondering what the best way to add it to our project is. We're excluding `node_modules/` from the JSX parsing for performance (don't want to pipe all that through Babel), which means that `var Selectable = require("react-selectable");` complains about unparsed JSX.
I tried doing `var Selectable = require("../../../../node_modules/react-selectable/dist/bundle.js");` but that pulls in the whole example app.

Do you think it'd be a good idea to provide a ready built JS file for people to use?

Thanks!

**UPDATE**: this seems like a good resource on this matter: http://chadly.net/2015/04/publishing-react-to-npm/

**Comment by lrecknagel** [comment]
I have the same issue, when I try to use it via:

```
import Selectable from 'react-selectable';
```

Can you please provide a solution for that. I read the article from dvdplm but I cant get it to work.

Kind Regards.

**Comment by unclecheese** [comment]
Yeah, I'm still trying to figure this one out. This is the first react/npm module I've released, so I'm a bit new to it.

**Comment by unclecheese** [comment]
Could this issue be solved by changing the `package.json` `"main": "src/react-selectable.js"` to point to `"dist/react-selectable.js"` instead?
